src/helpers.ts

```diff
@@ -61,15 +61,24 @@ function hasNodeExecutable(rootPath: string, executable: string): boolean {
  * @returns {string}
  */
 export function pathToJest(pluginSettings: IPluginSettings) {
-  const path = normalize(pluginSettings.pathToJest)
+  if (pluginSettings.pathToJest) {
```

Comment:
I removed the default setting for `pathToJest` From the package.json. I did this because I didn’t want to break any user configuration where they’d predefined it, but the default `pathToJest` defined before was incorrect for windows systems (forcing Windows users to explicitly define it). 

With `pathToJest` now potentially being null, I wanted to maintain old behaviour for linux/osx users, and my interpretation is that each code path used for the old version is equivalent in the new version, but for the windows scenario. In this particular line, I’m mapping the `path !== defaultPath` scenario for OSX users, which returned `normalize(pluginSettings.pathToJest)`.

Reply:
E.g. `path === defaultPath` => either `npm test —` or  `normalize(‘node_modules/.bin/jest’)`
`path !== defaultPath` => `normalize(path)`